rs/boundary_node/ic_boundary/src/core.rs

```diff
@@ -210,6 +230,76 @@ pub async fn main(cli: Cli) -> Result<(), Error> {
         cli.rate_limiting.rate_limit_generic.clone(),
     ));
 
+    // HTTP Logs Anonymization
+    let salt = Arc::new(ArcSwapOption::<Vec<u8>>::empty());
+
+    // Vault
+    let v = vault_from_config(
+        &CryptoConfig::new("/home/orricon/workspace/ic-cdk-logs/out/node-1/crypto".into()), // config
```

Comment:
Hmm? Not sure how crypto config looks like or if it has variables but we need to somehow make it not owned by `orricon` :) 

And, generally, move all this salt-related init stuff to some separate module or function ideally. So that it doesn't clog the `main()`.

Reply:
We should also make sure that the whole salt sharing process is not just started by default, but only when the corresponding config is provided over CLI/env variables (CryptoConfig, canister ID, etc.).

Reply:
Done, now the log anonymization canister ID _and_ the crypto config are both passed as CLI args and if either is not present then we don't configure log anonymization at all.